docs/DeveloperGuide.md

```diff
@@ -302,22 +302,23 @@ Given below is an example usage scenario and how the `ListAppointmentCommand` me
 **Step 1**: The user inputs `list_a` to list all appointments or `list_a pic\PATIENT_NRIC dic\DOCTOR_NRIC` to filter appointments.
 * The `list_a` command triggers `MediConnectParser#parseCommand(String)`, which identifies the command word and calls `ListAppointmentCommandParser#parse(String)` to handle the arguments.
 
-**Step 2**: The `ListAppointmentCommandParser#parse(String)` method checks for the presence of optional flags like `dic\` for doctor NRIC and `pic\` for patient NRIC. Based on these, it creates appropriate `Predicate` objects.
+**Step 2**: The `ListAppointmentCommandParser#parse(String)` method checks for the presence of optional flags like `dic\` for doctor NRIC and `pic\` for patient NRIC.
+ * Based on the presence of the doctor/patient's NRIC, it will affect whether their predicate `AppointmentEqualDoctorNricPredicate`/`AppointmentEqualPatientNricPredicate` is `PREDICATE_SHOW_ALL_DOCTORS`/`PREDICATE_SHOW_ALL_PATIENTS` or the predicate of the specified doctor/patient's NRIC.
```

Comment:
maybe can say `based on the presence of the doctor/patient NRIC flags`

Reply:
ok, thanks!